deblur/parallel_deblur.py

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+from __future__ import division
+
+from os.path import basename, join, split, splitext
+from os import mkdir
+
+from bfillings.sortmerna_v2 import build_database_sortmerna
+from skbio.parse.sequences import parse_fasta
+
+from qiime.parallel.util import ParallelWrapper
+
+
+class ParallelDeblur(ParallelWrapper):
```

Comment:
What's the rough rough balltime runtime estimate per sample for, say, 150nt @ 10,000 seqs per sample? Are there large memory requirements like > 8GB per sample?

We do per-sample parallelism on the AG data in the processing notebooks using `multiprocessing` so I think it's still relevant unless resource needs are prohibitive